src/api/system_api.js

```diff
@@ -58,6 +58,7 @@ module.exports = {
             auth: {
                 account: false,
                 system: false,
+                anonymous: true,
```

Comment:
Add check to create_system that fails if there is one system and `config.test_mode !== true`